ansible/roles/redpanda_broker/tasks/install-redpanda.yml

```diff
@@ -1,9 +1,16 @@
 ---
-- name: add the redpanda repo
-  ansible.builtin.shell: |
-    curl -1sLf {{ redpanda_repo_script }} | sudo -E bash
+- name: Download the Redpanda repo script
+  ansible.builtin.get_url:
+    url: "{{ redpanda_repo_script }}"
+    dest: "/tmp/redpanda_repo_script.sh"
+    mode: '0755'
```

Comment:
does this end up getting run on every invocation of ansible as well? The previous shell curl|bash thing didn't have a good way of handling only running if changes occurred, so I think it ran every run and updated the repo configs.

aside: we need to do this better, we should be doing curl|bash here as it won't work for airgapped ansible users. Better if we could just define the repo details directly.

Reply:
i think we'd need an airgap toggle to really support that mode, along with a testing environment that is built for airgap. i can't test it right now which means i can't build for it.